docs/breaking-changes.md

```diff
@@ -63,6 +63,29 @@ webContents.on('console-message', ({ level, message, lineNumber, sourceId, frame
 
 Additionally, `level` is now a string with possible values of `info`, `warning`, `error`, and `debug`.
 
+### Behavior Changed: `urls` property of `WebRequestFilter`.
+
+Previously, an empty urls array was interpreted as including all URLs. To explicitly include all URLs, developers should now use the `<all_urls>` pattern, which is a [designated URL pattern](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#all_urls) that matches every possible URL. This change clarifies the intent and ensures more predictable behavior.
+
+```js
+// Deprecated
+const deprecatedFilter = {
+  urls: []
+}
+
+// Replace with
+const newFilter = {
+  urls: ['<all_urls>']
+}
+```
+
+### Deprecated: `systemPreferences.isAeroGlassEnabled()`
```

Comment:
I think this section was backported incorrectly, let's remove it

Reply:
oh my bad, I missed this part, will update!

Reply:
actually this section was part of `35-x-y` so it's resolved correctly after I merged with the release branch and resolved the conflict!